catkin_home/src/navigation/base_control/scripts/cmd_vel.py

```diff
@@ -1,54 +1,52 @@
 #!/usr/bin/env python
-'''
-    Script that simulate navigation stack action of sending velocity commands.
-'''
 import rospy
 from geometry_msgs.msg import Twist
 
-def run_cmd_velocity():
-    '''
-        Main function that simultate the action of sending velocity commands.
-    '''
-    # Starts a new node.
```

Comment:
Keep a description for each Python/C++ file at the top, it is important to document this, so that future people understand its purpose, the same for functions.